frontend/server/src/main/java/com/amazonaws/ml/mms/util/ConfigManager.java

```diff
@@ -431,6 +441,14 @@ private int getIntProperty(String key, int def) {
         return Integer.parseInt(value);
     }
 
+    public String getDefaultResponseTimeout() {
+        return prop.getProperty(DEFAULT_RESPONSE_TIMEOUT, "2");
+    }
+
+    public void setDefaultResponseTimeout(String timeout) {
```

Comment:
We don't need a setter.

Reply:
We would need the setter because this can come through CLI.